factory/.env

```
@@ -8,25 +8,25 @@
 BASE_IMAGE='debian:12-slim'
 
 # Node Versions: https://nodejs.org/en/download/releases/
-FACTORY_DEFAULT_NODE_VERSION='20.13.1'
+FACTORY_DEFAULT_NODE_VERSION='20.14.0'
 
 # Node Versions: https://nodejs.org/en/download/releases/
 NODE_VERSION="${FACTORY_DEFAULT_NODE_VERSION}"
 
 # Update this to deploy the docker factory if you make changes to factory.Dockerfile or install scripts
-FACTORY_VERSION='4.0.1'
+FACTORY_VERSION='4.0.2'
 
 # Chrome versions: https://www.ubuntuupdates.org/package/google_chrome/stable/main/base/google-chrome-stable
-CHROME_VERSION='125.0.6422.60-1'
+CHROME_VERSION='125.0.6422.141-1'
 
 # Cypress versions: https://www.npmjs.com/package/cypress
-CYPRESS_VERSION='13.10.0'
+CYPRESS_VERSION='13.11.0'
 
 # Edge versions: https://packages.microsoft.com/repos/edge/pool/main/m/microsoft-edge-stable/
-EDGE_VERSION='125.0.2535.51-1'
+EDGE_VERSION='125.0.2535.79-1'
```

```suggestion
EDGE_VERSION='125.0.2535.85-1'
```
Released today and Ubuntu has already updated my local machine with this version.

---

I had issues pulling it from source but I will give it a try again!

---

It's very new, so I can imagine there may be problems. I did manage to grab it using the cypress/factory process though:

```text
$ docker run -it --rm test
root@c77816b01b07:/opt/app# microsoft-edge --version
Microsoft Edge 125.0.2535.85 
root@c77816b01b07:/opt/app# 
```